js_tutorial/exercises/002_arithmetic.js

```diff
@@ -0,0 +1,69 @@
+// Addition of two numbers
+
+let a , b;
```

Comment:
You're not using this initialized a,b variables.
There is no need of  " let a,b; "